package.json

```diff
@@ -14,9 +14,9 @@
 		"build": "babel app.js --out-dir build",
 		"dev": "NODE_ENV=development nodemon  --require @babel/register app.js",
 		"coverage": "nyc report --reporter=text-lcov | coveralls",
-		"dbInit-dev": "NODE_ENV=development sequelize db:create || NODE_ENV=development sequelize db:migrate",
-		"dbInit-test": "NODE_ENV=test npx sequelize db:create || NODE_ENV=test npx sequelize db:migrate",
-		"dbInit": "npm run dbInit-dev || npm run dbInit-test"
+		"dbInit-dev": "NODE_ENV=development sequelize db:create && NODE_ENV=development sequelize db:migrate",
```

Comment:
Hey @coolbeatz71 why did you change these lines, we chose `||`so that when the db is already created the migration doesn't fail. when using `&&` the creation of db must be successful for the migration to run

Reply:
Yup @dusmel. I understand your concern. I changed it because, in case we need to prepare at the same time the test and dev DB, this oblige us to run the script twice. Because of this `||`. But I also noticed that using `&&` is preventing the script to finish in the case where the DB already exist. Leading us to an error. The solution for this issue is to use `&` which has the advantage of running both script simultaneously and when the DB already exist, it is not preventing the rest of the script to be executed.

### **See this screenshot**
<img width="870" alt="Screen Shot 2019-05-07 at 17 35 04" src="https://user-images.githubusercontent.com/25999336/57312822-940e4000-70ee-11e9-93bc-c3d3ea282a11.png">